automation/DeployToHPVS.bashlib

```diff
@@ -489,15 +489,15 @@ config_file_wizard(){
     # Config variable: APIKey                                                  #
     #--------------------------------------------------------------------------#
     PROMPT_USER_READARGS="-s"
-    prompt_user "Enter an IAM key (platform API key) for your IBM Cloud account" false
+    prompt_user "Enter an IAM key (platform API key) for your IBM Cloud account. An example API key looks like this k-y2lsNxfeKbaidcCG6kQVypLDR_WjmDPe-2Uuq_PTYx" false
     unset PROMPT_USER_READARGS
     APIKey=${USERINPUT}
     change_config_var ${1} "APIKey" "${APIKey}"
     
     #--------------------------------------------------------------------------#
     # Config variable: registryURL                                             #
     #--------------------------------------------------------------------------#
-    prompt_user "Enter URL of container repository" false "docker.io"
+    prompt_user "Enter URL of container repository. For example, docker.io for Dockerhub or us.ibm.cr for IBM Cloud Container Registry" false "docker.io"
     registryURL=$(echo "${USERINPUT}" | tr '[:upper:]' '[:lower:]') # Make lowercase
```

Comment:
the url for the container registry should be this `us.icr.io` and add a line pointing them to the documentation

"For more information about creating a container registry in IBM Cloud see here https://cloud.ibm.com/apidocs/container-registry"

Also I think it should be called container registry not repository, let me know what you think about that